tests/test_with_oauth.rs

```diff
@@ -716,19 +717,18 @@ async fn test_playlist_remove_all_occurrences_of_tracks() {
 #[ignore]
 async fn test_playlist_remove_specific_occurrences_of_tracks() {
     let playlist_id = Id::from_id("5jAOgWXCBKuinsGiZxjDQ5").unwrap();
-    let tracks = vec![
-        TrackPositions::new(
-            Id::from_uri("spotify:track:4iV5W9uYEdYUVa79Axb7Rh").unwrap(),
-            vec![0, 3],
-        ),
-        TrackPositions::new(
-            Id::from_uri("spotify:track:1301WleyT98MSxVHPZCA6M").unwrap(),
-            vec![7],
-        ),
-    ];
-    oauth_client()
-        .await
-        .playlist_remove_specific_occurrences_of_tracks(playlist_id, tracks, None)
+    let track_position_0_3 = TrackPositions::new(
+        Id::from_uri("spotify:track:4iV5W9uYEdYUVa79Axb7Rh").unwrap(),
+        vec![0, 3],
+    );
+    let track_position_7 = TrackPositions::new(
+        Id::from_uri("spotify:track:1301WleyT98MSxVHPZCA6M").unwrap(),
+        vec![7],
+    );
+    let tracks = vec![&track_position_0_3, &track_position_7];
+    oauth_client()
+        .await
+        .playlist_remove_specific_occurrences_of_tracks(&playlist_id, tracks, None::<&str>)
```

Comment:
`playlist_id` shouldn't need the `&`, just like how a `&str` initialization doesn't need `&` when passed afterwards.